string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-vars */
+function isLowerCased(word) {
+  let letterCheck = '';
+  for (let i = 0; i < word.length; i++) {
+    letterCheck += word[i].toLowerCase();
+  }
+  if (letterCheck[0] === word[0] && letterCheck[1] === word[1]) {
+    return true;
+  } if (letterCheck[0] !== word[0]) {
+    return false;
+  } else if (letterCheck[0] === word[0] && letterCheck[1] !== word[1]) {
```

Comment:
It's not entirely clear to me what these mean. I think you are making your function too specific for the inputs. You can try a couple of strategies:

if a string is `===` to another string, they are equal. so you could lowercase the input and compare the lowercased copy to the original

otherwise, you could loop through the entire string and compare each character of the original to a lowercased version of that character.